Resources/NetHookAnalyzer2/NetHookAnalyzer2/Specializations/CSGO/CSGOSOMultipleObjectsGCSpecialization.cs

```diff
@@ -3,7 +3,8 @@
 using System.IO;
 using ProtoBuf;
 using ProtoBuf.Meta;
-using SteamKit2.GC.CSGO.Internal;
+//using SteamKit2.GC.CSGO.Internal;
+using SteamKit2.GC.Dota.Internal; // CSGO no longer has objects_added and objects_removed
```

Comment:
if CS:GO no longer has those fields, should we just delete the code in this class that handles them? Or is data still there but it just has no field in the protos?

Reply:
Last issue outstanding.